caikit/runtime/service_generation/core_module_helpers.py

```diff
@@ -45,6 +45,7 @@ def get_module_info(ck_module: Type[ModuleBase]) -> Optional[ModuleInfo]:
     2. The module derives from a base class that itself derives from one of the
         known type-hierarchy derived from `ModuleBase`.
     """
+    # NOTE: all of this assumes <library> has no .
```

Comment:
https://github.com/caikit/caikit/issues/30

Reply:
Yeah, this bit is used mostly to determine the `task` of a model which I would love to put in the module decorator instead of trying to inspect from the module path